test/500_weave_multi_cidr_test.sh

```diff
@@ -42,36 +42,41 @@ assert_bridge_cidrs() {
 
 start_suite "Weave run/start/attach/detach with multiple cidr arguments"
 
-weave_on $HOST1 launch
+weave_on $HOST1 launch -debug -iprange 10.2.3.0/24
 weave_on $HOST1 launch-dns 10.254.254.254/24
 
 # Run container with three cidrs
-CID=$(start_container $HOST1 10.2.1.1/24 10.2.2.1/24 10.2.3.1/24 --name=multicidr -h $NAME | cut -b 1-12)
+CID=$(start_container $HOST1 10.2.1.1/24 10.2.2.1/24 --subnet 10.2.3.0/24 --name=multicidr -h $NAME | cut -b 1-12)
 assert_container_cidrs $HOST1 $CID 10.2.1.1/24 10.2.2.1/24 10.2.3.1/24
 assert_zone_records $HOST1 $CID $NAME. 10.2.1.1 10.2.2.1 10.2.3.1
 
 # Remove two of them
-weave_on $HOST1 detach 10.2.1.1/24 10.2.3.1/24 $CID
+weave_on $HOST1 detach 10.2.1.1/24  --subnet 10.2.3.0/24 $CID
 assert_container_cidrs $HOST1 $CID 10.2.2.1/24
```

Comment:
hmm. you really want `assert_container_cidrs` to check that *exactly* those addresses are present, don't you? Which it doesn't.